### ISSUE TITLE
GetTeam is actually LookupTeam

### ISSUE THREAD

**Issue by ghostsquad** [pulumi/pulumi-github#199]
## Hello!

- Vote on this issue by adding a 👍 reaction
- To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already)

## Issue details

The ApiDocs specify `GetTeam` as the function, but once you look at that page, it's actually `LookupTeam`.

https://www.pulumi.com/registry/packages/github/api-docs/getteam/

**Comment by guineveresaenger** [comment]
Hi @ghostsquad - thank you for reporting this issue!

I presume you're using the Go SDK? Because we generate for multiple languages at once, and because of the way the API docs are generated, sometimes there's discrepancies.

We do make a note of this here:

<img width="677" alt="Screen Shot 2022-03-21 at 11 52 07 AM" src="https://user-images.githubusercontent.com/13116240/159343693-f9c98d86-9436-40d8-964d-b28216754278.png">

Apologies for any confusion!

**Comment by ghostsquad** [comment]
Yep. This is exactly the discrepancy. Since the note is hidden inside the lodge describing GetTeam, it's harder to find/identify from the docs.
